### ISSUE TITLE
Remove this repository? These docs are deprecated/outdated.

### ISSUE THREAD

**Issue by JoshuaRLi** [standardnotes/doc#4]
Also [this](https://github.com/standardnotes/desktop-beta), which appears to be empty.

**Comment by mobitar** [comment]
Yeah..the dev docs are all over the place currently. Really need to clean it all up. Not too happy with the system for docs.standardnotes.org. But for this repo, there are links to here from here: https://standardnotes.org/developers

**Comment by JoshuaRLi** [comment]
Haha yeah, I'm kind of a stickler for neatness. Just good knowing you're aware.
